### DISCUSSION TITLE
Bun v1.0.35

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#9568] [Announcements]
Bun v1.0.35 fixes a regression introduced in Bun v1.0.34.

To install Bun v1.0.35:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.35:
```bash
bun upgrade
```

### **[Read Bun v1.0.34's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.34)**

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.35'>Bun v1.0.35</a>.</em>

**Comment by TeddyHuang-00** [comment]
The release note link is pointing to last release?

**Reply by ElectricS01** [reply]
It seems 1.0.35 was too small for them to bother making a blog post for it. Note that https://bun.sh/blog/bun-v1.0.35 doesn't exist

**Reply by w1ldy0uth** [reply]
yeah, it's just a fix
